packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.tsx

```diff
@@ -134,10 +154,19 @@ export function ComboboxContent(props: ComboboxContentProps): JSX.Element {
 function Search(props: {
   placeholder?: string;
   searchValue: string;
+  open: boolean;
   setSearchValue: Dispatch<SetStateAction<string>>;
 }): JSX.Element {
   const searchRef = useRef<HTMLInputElement>(null);
 
+  useEffect(() => {
+    if (props.open) {
+      setTimeout(() => {
```

Comment:
required to focus the search input on open. you cannot `focus()` properly on something that has the css property `visibility: hidden`

what seems to be happening is a race condition where we trigger the open, focus the search and then add the css class that changes the visibility

adding this timeout of 0 moves this operation down to the bottom of the call stack or whatever it's called for the JS event loop, giving it enough time to do its thing